### ISSUE TITLE
How to run as non-root user?

### ISSUE THREAD

**Issue by HenriWahl** [accetto/ubuntu-vnc-xfce-firefox-plus#2]
Hi,
cool image, but how exactly am i supposed to run as non-root user inside the container? The information at https://github.com/accetto/ubuntu-vnc-xfce/wiki/how-to#how-to-override-vnc-user does not really help. I just get

`root@sserver ~# docker run -it --rm -p 8081:6901 --user 2017:2000 accetto/ubuntu-vnc-xfce` 
`USER_ID: 2017, GROUP_ID: 2000`
`nss_wrapper location: /usr/lib/libnss_wrapper.so`
` `
`------------------ change VNC password  ------------------`
`mkdir: cannot create directory ‘/home/headless/.vnc’: Permission denied`

How exactly is it meant to be used?

**Comment by accetto** [comment]
Thank you for your feedback. It seems like something have changed. There are certainly some limitations when changing users, but in your case it should work. I'll look at it.

*Remark*

This issue actually belongs to the base container [accetto/ubuntu-vnc-xfce](https://github.com/accetto/ubuntu-vnc-xfce), so I've filed the [issue #8](https://github.com/accetto/ubuntu-vnc-xfce/issues/8) there.

However, creating containers from **this** image like

```bash
docker run -d -P --user 2017:2000 --rm accetto/ubuntu-vnc-xfce-firefox-plus
```

or

```bash
docker run -it -P --user 2017:2000 accetto/ubuntu-vnc-xfce-firefox-plus bash
```

causes the following error:

```bash
cp: failed to access '/home/headless/.mozilla/firefox': Permission denied
```

**Comment by accetto** [comment]
Fixed in the release 19.05.
